### DISCUSSION TITLE
Share post announcing removal of some APIs

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#107] [Topic Ideas]
Pulling this out of this PR: https://github.com/WordPress/gutenberg/pull/50240 I think it would be a great use of the developer blog to announce what's being removed and raise awareness here, ideally perhaps with 1on1 outreach to folks impacted.

**Comment by justintadlock** [comment]
We have a "Breaking changes and deprecations" section in the What's new for developers? monthly roundup. If this makes it in this week, we can include it in the [May 2023 edition](https://github.com/WordPress/developer-blog-content/issues/101) ([Google Docs link](https://docs.google.com/document/d/14Kz5W5j2I53bAqe7DzUurTKD1Ey7Oc6nAnggWrAXQNw/edit#)). If not, it could make it into the June 2023 edition.

This topic could certainly have its own post if this is big enough to warrant it too. CC: @ryanwelcher, @mburridge - Any input on this?

**Comment by mburridge** [comment]
I'm not sure that this warrants a full post. I think a section in the What's new posts should be sufficient, perhaps with periodic reminders in the posts about the upcoming deprecation.

**Comment by annezazu** [comment]
Awesome! I trust y’all’s read. Can we keep this open to keep track of where
it’s mentioned? It would be great to consolidate here.

On Wednesday, May 3, 2023, Michael Burridge ***@***.***>
wrote:

> I'm not sure that this warrants a full post. I think a section in the
> What's new posts should be sufficient, perhaps with periodic reminders in
> the posts about the upcoming deprecation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/WordPress/developer-blog-content/discussions/107#discussioncomment-5789112>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGN7BE6OGXGBYVYLYZQM2YLXEIN5FANCNFSM6AAAAAAXTLJGYY>
> .
> You are receiving this because you authored the thread.Message ID:
> <WordPress/developer-blog-content/repo-discussions/107/comments/5789112@
> github.com>
>


-- 
*Anne McCarthy*
WordPress Product Liaison
Pronouns
<https://www.hrc.org/resources/talking-about-pronouns-in-the-workplace>:
she/her/hers | they/them/theirs

**Comment by bph** [comment]
Deprecations are also part of the WordPress Dev Notes. So they should be covered for all. :-)
